crane/cli.go

```diff
@@ -25,8 +25,8 @@ var (
 	).Short('p').OverrideDefaultFromEnvar("CRANE_PREFIX").String()
 	excludeFlag = app.Flag(
 		"exclude",
-		"Exclude group or container",
-	).Short('e').OverrideDefaultFromEnvar("CRANE_EXCLUDE").String()
+		"Exclude group or container. Can be repeated.",
+	).Short('e').OverrideDefaultFromEnvar("CRANE_EXCLUDE").PlaceHolder("container|group").Strings()
```

Comment:
FYI, kingpin doesn't support multiple values from an env variable, so the repeated feature will only be available via the CLI flag